tests/loaders/test_https_loader.py

```diff
@@ -8,33 +8,31 @@
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
-from os.path import abspath, join, dirname
+from os.path import abspath, dirname, join
 
-from preggy import expect
-import mock
-# from tornado.concurrent import Future
 import tornado.web
-from tests.base import PythonTestCase, TestCase
-from tornado.concurrent import Future
+from preggy import expect
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'preggy'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)